package.json

```diff
@@ -63,7 +63,7 @@
   "scripts": {
     "build": "npm run clean && tsc --project tsconfig.build.json && npm run copy-files",
     "clean": "rimraf build/",
```

Comment:
hi @narekhovhannisyan I've added this line to copy the _html-export-template.html_ file to the _build_ directory on compilation (tsc), any idea why it doesn't work? :) any tip would be appreciated

Reply:
seems working locally